tipg/resources/response.py

```diff
@@ -1,15 +1,42 @@
 """tipg custom responses."""
 
-from fastapi.responses import ORJSONResponse
+import decimal
+from typing import Any
 
+import orjson
 
-class GeoJSONResponse(ORJSONResponse):
+from fastapi.responses import JSONResponse
+
+
+def default(obj):
+    """Instruct orjson what to do with types it does not natively serialize"""
+    if isinstance(obj, decimal.Decimal):
+        return str(obj)
+
+
+class GeoJSONResponse(JSONResponse):
     """GeoJSON Response"""
 
     media_type = "application/geo+json"
 
+    def render(self, content: Any) -> bytes:
+        """Render the content into a JSON response using orjson"""
+        return orjson.dumps(
+            content,
+            default=default,
+            option=orjson.OPT_NON_STR_KEYS | orjson.OPT_SERIALIZE_NUMPY,
+        )
```

Comment:
could we create a custom JSONResponse class to avoid duplicating the `render` code?

Reply:
Yeah should be easy

Reply:
Done :slightly_smiling_face:

Reply:
Looks good to me. Can you check if we need to do the same thing for postgres fields of type "numeric" as well (which is equivalent to the postgres "decimal" type?